### DISCUSSION TITLE
Thanks for an amazing port

### DISCUSSION THREAD

**Discussion by darkfader7721** [Novum/vkQuake#424] [General]
I highly enjoy the Vulkan API.
I'm curious if you ever plan on adding any mod support?

Traditional Source Ports were were able to utilize .pak or .pk3 files for mods, is this something you plan on implementing?

Thanks,

**Comment by Novum** [comment]
I don't understand what you're asking. There has been mod support in any Quake engine since day 1?

**Reply by darkfader7721** [reply]
I'm used to dropping .pk3 archives into the /id1 folder to overwrite existing files.
So, if I want to modify the textures, or the hud ... what file type should my mod be so vkQuake recognizes it?

Thanks for the response, Novum.

**Reply by VGkav** [reply]
https://quakewiki.org/wiki/Installing_addons

**Reply by darkfader7721** [reply]
I'm familiar with installing addons such as -ad (Arcane Dimensions) or -alk1.1 (Alkaline), which works great for vkQuake, or even community made maps.

Forgive my verbiage if it doesn't fit, but I'm looking to modify vkQuake with purely .pk3 archives (traditional darkplaces / ezQuake mods) with modern image support. (.png instead of traditional .lmp) ... if that makes sense. Quakespasm recognizes and loads .pk3 files, but vkQuake doesn't recognize them. Even if I create an 'addon', the files have to be recognized by the Source Port, in which it seems vkQuake only recognizes traditional files archived within the original Quake .pak files.

I apologize if my terms or verbiage is lacking, I am a Quake fan from the 2000's only casually familiar with modding, and likely used to the modding support that only darkplaces / ezQuake / Quakespasm provides.

Thanks.
